src/Core/Repositories/EntityFramework/OrganizationRepository.cs

```diff
@@ -68,7 +80,7 @@ public async Task UpdateStorageAsync(Guid id)
                 var dbContext = GetDatabaseContext(scope);
                 var ciphers = await dbContext.Ciphers
                     .Where(e => e.UserId == null && e.OrganizationId == id).ToListAsync();
-                var storage = ciphers.Sum(e => e.AttachmentsJson?.RootElement.EnumerateArray()
+                var storage = ciphers.Sum(e => JsonDocument.Parse(e.Attachments)?.RootElement.EnumerateArray()
```

Comment:
So, would `e.Attachments` ever be `== "null"` (null token string) and therefore `JsonDocument.Parse()` would return `null`? OR, if `e.Attachments == null`, would `JsonDocument.Parse()` throw an exception instead? Seems from the docs it doesn't throw `ArgumentNullException`, however it could throw `JsonException` with the message, `"json does not represent a valid single JSON value."`

Reply:
addressed with a null check on attachments, though this method has been moved to the base repo